shacl12-core/index.html

```diff
@@ -3435,9 +3435,15 @@ <h4>sh:nodeKind</h4>
 								The node kind (IRI, blank node, literal, triple term or combinations of these) of all value nodes.
 								<span data-syntax-rule="nodeKind-maxCount">A shape has at most one value for <code>sh:nodeKind</code>.</span>
 								The value of sh:nodeKind in a shape is either an IRI or a blank node that is a well-formed SHACL list where all members are IRIs.
-								<span data-syntax-rule="nodeKind-in">The values or members of <code>sh:nodeKind</code> in a shape are one of the following seven instances of the class <code>sh:NodeKind</code>:
-								<code>sh:BlankNode</code>, <code>sh:IRI</code>, <code>sh:Literal</code> 
-								<code>sh:BlankNodeOrIRI</code>, <code>sh:BlankNodeOrLiteral</code>, <code>sh:IRIOrLiteral</code> and <code>sh:TripleTerm</code>.</span>
+								<br /><br />
+								<span data-syntax-rule="nodeKind-in">
+									If the values of <code>sh:nodeKind</code> are IRIs, then the values of <code>sh:nodeKind</code> in a shape are one of the following seven instances of the class <code>sh:NodeKind</code>:
+									<code>sh:BlankNode</code>, <code>sh:IRI</code>, <code>sh:Literal</code>
+									<code>sh:BlankNodeOrIRI</code>, <code>sh:BlankNodeOrLiteral</code>, <code>sh:IRIOrLiteral</code> and <code>sh:TripleTerm</code>.
+									<br /><br />
+									If the values of <code>sh:nodeKind</code> are well-formed SHACL lists, then the values of <code>sh:nodeKind</code> in a shape are one of the following four instances of the class <code>sh:NodeKind</code>:
```

Comment:
I think you mean the "members of those lists", not the "values".

Reply:
Thanks! Fixed it.